qcodes_contrib_drivers/drivers/Lakeshore/Model_625.py

```diff
@@ -0,0 +1,524 @@
+import logging
+import time
+from typing import Union, Tuple
+
+
+from qcodes import VisaInstrument
+from qcodes.utils.validators import  Numbers, Enum
+
+log = logging.getLogger(__name__)
+
+class Lakeshore625(VisaInstrument):
+    """
+    Driver for the Lakeshore Model 625 superconducting magnet power supply.
+
+    This class uses T/A and A/s as units.
+
+    Args:
+        name (str): a name for the instrument
+        address (str): VISA address of the device
+        current_ramp_limit: A current ramp limit, in units of A/s
+    """
+
+    def __init__(self, name: str, coil_constant: float,  field_ramp_rate: float, address: str,
+                 reset: bool=False, terminator:str='', **kwargs) -> None:
+
+        super().__init__(name, address, terminator=terminator, **kwargs)
+    
+        # Add reset function
+        self.add_function('reset', call_cmd='*RST')
+        if reset:
+            self.reset()
+
+        # Add power supply parameters
+        self.add_parameter(name='current_limit',
+                           unit="A",
+                           set_cmd=lambda x: self._set_curent_limit(x),
```

Comment:
I don't think you need the lambda functions here. As far as I can see `set_cmd=self._set_curent_limit` should be equivalent

Reply:
Same in the parameters below

Reply:
I guess so. Don't know why some parameter set functions where like this. I guess I have copied from several other drivers... Thanks for spotting this.